src/server/util/insertData.js

```diff
@@ -228,7 +228,7 @@ async function insertStandardConversions(conn) {
 			sourceName: 'BTU',
 			destinationName: 'm³ gas',
 			bidirectional: true,
-			slope: 9.625,
+			slope: 2.73e-5,
```

Comment:
How could we potentially prevent issues with conversion rates between units. i cant seem to find libraries for it, but it feels like issues with conversion could be potentially hard to debug.

Reply:
It is hard to know the correct value. I think OED needs to be very careful with the standard units provided to sites to make sure they are accurate. I don't know of a good alternative.